packages/ui/src/components/primitives/Footer.tsx

```diff
@@ -14,6 +14,10 @@ const navigation = {
 		{ name: 'Cookies', href: 'https://www.answeroverflow.com/cookies' },
 		{ name: 'EULA', href: 'https://www.answeroverflow.com/eula' },
 		{ name: 'Docs', href: 'https://docs.answeroverflow.com' },
+		{
+			name: 'Contributors',
```

Comment:
lets add browse communities to the footer, might need categories on the footer now

Reply:
:+1: - I'll make an issue for categories on the footer

Reply:
can we just do it in this PR? its pretty tiny so may as well get it out the way